modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/ColumnParams.java

```diff
@@ -180,7 +180,7 @@ public Builder scale(int scale) {
          *
          * @return {@code this}.
          */
-        public Builder length(int length) {
+        public Builder length(Integer length) {
```

Comment:
Please add `@Nullable` for method `org.apache.ignite.internal.catalog.commands.ColumnParams#length` and specify in documentation what `null` means.